### ISSUE TITLE
Shared storage

### ISSUE THREAD

**Issue by xloem** [radi-cho/datasetGPT#1]
It would be good if the community would curate one large dataset for people to share and contribute to together.

**Comment by radi-cho** [comment]
Agree. We can take insights from the Alpaca paper and related works as a starting point to crowdsource open datasets. They could be instruction-following, task-specific e.g. math problem reasoning, QAs, or generic texts to compare existing models.

**Comment by radi-cho** [comment]
Considering a "hub" where datasets generated by the tool can be gathered. Contributions (or ideas) are welcome.
